datacommons_pandas/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## 0.0.1
+
+**Date** - 08/24/2020
+
+**Release Tag** - [pd.0.0.1](https://github.com/datacommonsorg/api-python/releases/tag/pd0.0.1)
+
+**Release Status** - Current head of branch [`master`](https://github.com/datacommonsorg/api-python/tree/master)
+
+Added pandas wrapper functions.
+
+-   `build_time_series` will construct a pd.Series for a given StatisticalVariable and Place, where dates are the index for the time series.
```

Comment:
where the time series are indexed by date

Reply:
done